skbeam/core/accumulators/binned_statistic.py

```diff
@@ -70,8 +70,13 @@ def __init__(self, sample, statistic='mean',
             A sequence of lower and upper bin edges to be used if the
             edges are not given explicitely in `bins`. Defaults to the
             minimum and maximum values along each dimension.
-
+        mask : array_like
+            array of ones and zeros with the same shape as `sample`.
+            Values with mask==0 will be ignored.
         """
+        if mask is None:
+            mask = np.ones_like(sample)
```

Comment:
I don't understand (but I may be wrong! :-) ). `sample` are the bins. How can `mask` be the same shape as `sample` (i.e. for 2D bins, then the shape of `sample` will be (2, dimx, dimy) where dimx, dimy are dimensions of sample)?
How are you testing the code? Are you using real data? I think for something like this, we should have a working example visually tested and posted before accepting this, regardless of the tests.

I can also write a notebook later using your code, see if it works, if it helps :-).